client/apps/game/src/dojo/queries.ts

```diff
@@ -399,3 +401,82 @@ export const getMapFromTorii = async <S extends Schema>(
     false,
   );
 };
+
+export const getQuestTilesFromTorii = async <S extends Schema>(
```

Comment:
why needed ?

Reply:
We are syncing quest tiles (model) so the data can be pulled from recs when the `system-manager` recognises a tile update. This data is mainly used in the `quest-manager` to add quests on the manager update:

```
    this.quests.addQuest(
      entityId,
      structureType,
      id,
      game_address,
      position,
      level,
      resource_type,
      amount,
      capacity,
      participant_count,
    );
    ```
    
For now this may be redundant based on recent changes to how tiles are not read from recs but are fetched, but thinking further ahead we may need this to differentiate between a DS game vs other game unless that info is included in the `Tile` model itself.